development/Vagrantfile.example

```diff
@@ -1,20 +1,72 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-BOX = ENV.fetch('BOX', 'ubuntu-12.04')
+require 'pry'
 
-Vagrant.require_plugin 'vagrant-proxyconf'
+BOX = ENV.fetch('BOX', 'ubuntu/bionic64')
+DISABLE_SHARE = false
+ENABLE_PROXY = false
+
+DOCKER_SOCKET="/var/run/docker.sock"
```

Comment:
Missing spaces around the `=`

Reply:
Resolved via: 6f14c17e0239fcc92ab3881d317fc69d60b5e774